taxonomies/packaging_materials.txt

```diff
@@ -357,11 +372,13 @@ hu:Pamut, 60 COT
 it:Cotone
 nl:Katoen
 ru:Хлопок
+wikidata:en:Q8231603
```

Comment:
Would https://www.wikidata.org/wiki/Q11457 better fit?

Reply:
I reviewed all of them, good job. Just a little issue with "cotton", see my previous comment.

Reply:
I'm not sure about Q11457, it's the raw material, the other one is a fabric made out of cotton which is more suitable for wrapping products (I guess).